### ISSUE TITLE
[init/scaffold] feedback on working directory and created files

### ISSUE THREAD

**Issue by filtercake** [pingyhq/pingy-cli#73]
```
Scaffolding hello name
its a me

✔ Created package.json
? Do you want Pingy to scaffold the following files for you? 
 
  ├── index.pug
  └─┬ components/
    ├── button.pug
    └── _nuclides.sass
  
(Y/n) 
```

this would be a good place to remind in what directory i am initing to so i don't accidentally use the wrong one :)

`? Do you want Pingy to scaffold the following files for you into $current_directory?`

also, even if i remember to say no here to abort, `pingy.json` and `package.json` are already created. for now it would just be good to be noted about that so i don't leave a dirty folder behind.

a later version could:
- not create any files before i confirm anything
- instead of (Y/n) give a select like
  ```
  ? You are about to init/scaffold into $current_directory
  ❯ yes, that is correct
     no, let me create a new one
  ```

**Comment by davej** [comment]
Thanks for your feedback. Here is what I am thinking:
![image](https://user-images.githubusercontent.com/26971/33212829-ac6fd4ec-d11c-11e7-8ab2-ff3bbb0a5f55.png)

A couple of notes:
* Current directory is now shown in the file tree
* `package.json` and `pingy.json` are now created after the scaffold step so if you choose to abort then Pingy will not have created any files.

I might add the option to create a new directory in the future  but not for now.

Any thoughts?

**Comment by filtercake** [comment]
Awesome 👍🚀

**Comment by davej** [comment]
Published in `v0.10.4`
